cypress/integration/work-order/create/create.spec.js

```diff
@@ -203,16 +201,18 @@ describe('Raise repair form', () => {
         'INP5R001 - Pre insp of wrks by Constructr'
       )
       // Does not autopopulate priority description
-      cy.get('#priorityDescription').should('have.value', '')
+      cy.get('#priorityCode').should('have.value', '')
 
       // Select SOR code with priority attached
       cy.get('input[id="rateScheduleItems[0][code]"]')
         .clear()
         .type('DES5R003 - Immediate call outs')
+
       // Autopopulates priority description
-      cy.get('#priorityDescription').should('have.value', '1 [I] IMMEDIATE')
+      cy.get('#priorityCode')
```

Comment:
shouldn't we check here if `#priorityCode` has value '`1 [I] IMMEDIATE'`
something like : `cy.get('#priorityCode')
        .find('option:selected')
        .should('have.text', '1 [I] IMMEDIATE')`

Reply:
👍